### PR TITLE
Feat(bank view) data integration

### PR DESCRIPTION
**Pull request by CaioVicenzi**
### Issue

Finishing the bank account detailing.

### Overview

This PR aims to finish doing my BankView task, the detailing of the BankAccount and the editing of its name, as well as the deletion, its goal is to fulfill all these objectives.

### Key Changes

- **<change-in-files>**: I was using the wrong file to mount this view, but I changed it to the correct file now.

- **total-calculation**: There is now a calculation behind the total money in all accounts of the financial institution's account.

- **back-button**: Now the back button only comes back with a warning if the name is different from the one saved in the bank.

### CODE REVIEW

**Comment by giovannifavorin** [comment]
that view isn't connected to the app navigation

**Review by giovannifavorin — CHANGES_REQUESTED** [review]
Connect that view to the app navigation

File: MoneyManagement App/View/Shared/Rows/Header.swift [hunk 1]
```diff
@@ -0,0 +1,35 @@
+//
+//  Header.swift
+//  MoneyManagement App
+//
+//  Created by Caio Marques on 07/11/24.
+//
```
Comment (RafaelOsiro) [inline MoneyManagement App/View/Shared/Rows/Header.swift:6]: Please remove the comments between lines 1 to 6

File: MoneyManagement App/View/Views/Finances/BankView/BankViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,98 @@
+//
+//  BankViewModel.swift
+//  MoneyManagement App
+//
+//  Created by Caio Marques on 10/11/24.
+//
```
Comment (RafaelOsiro) [inline MoneyManagement App/View/Views/Finances/BankView/BankViewModel.swift:6]: Please remove the comments between lines 1 to 6

File: MoneyManagement App/View/Views/Finances/Components/Balance.swift [hunk 3]
```diff
@@ -0,0 +1,38 @@
+//
+//  Balance.swift
+//  MoneyManagement App
+//
+//  Created by Caio Marques on 07/11/24.
+//
```
Comment (RafaelOsiro) [inline MoneyManagement App/View/Views/Finances/Components/Balance.swift:6]: Please remove the comments between lines 1 to 6

**Review by RafaelOsiro — CHANGES_REQUESTED** [review]
You have a conflict in your branch.

Please update your branch with develop and solve those comments that I leave in my Review.
If you have something to discuss, please start a discussion in the comment

**Pull request closed without merging**